tools/0a-multi-lpar-preseed/templates/preseed.cfg.jinja

```diff
@@ -0,0 +1,134 @@
+
+# installation source location
+d-i mirror/protocol string http
+d-i mirror/http/hostname string us.ports.ubuntu.com
+d-i mirror/http/directory string /
+d-i mirror/http/proxy string
+d-i mirror/suite string xenial
+
+# install stuff
+d-i live-installer/enable boolean false
+d-i tasksel/first multiselect standard system utilities, OpenSSH server, Basic Ubuntu server
+
+# late_commands
+# this is where the magic happens....
+d-i preseed/late_command string \
+# activate 2nd interface and set to layer 2 bridge
+in-target chzdev --no-root-update -pVe {{ OSA2_DEVNO }} bridge_role=primary; \
+in-target chzdev -p {{ OSA2_DEVNO }} layer2=1; \
+# configure 2nd interface for multiple vlans by stuffing the config manually
+# Note: the following is for POK Solution Test -- custom private networking
+#touch /target/etc/network/interfaces.d/private_int; \
+#echo -e "auto {{ OSA2_ENCNO }}" > /target/etc/network/interfaces.d/private_int; \
+#echo -e "iface {{ OSA2_ENCNO }} inet manual" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "auto {{ OSA2_ENCNO }}.1295" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "iface {{ OSA2_ENCNO }}.1295 inet static" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        address {{ OSA2_ADDRESS }}" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        netmask 255.255.255.0" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        network 10.20.95.0" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        broadcast 10.20.95.255" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        vlan-raw-device {{ OSA2_ENCNO }}" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        up ip route add 10.20.0.0/16 via 10.20.95.254 || true" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "auto {{ OSA2_ENCNO }}.1299" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "iface {{ OSA2_ENCNO }}.1299 inet manual" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        vlan-raw-device {{ OSA2_ENCNO }}" >> /target/etc/network/interfaces.d/private_int; \
+# get the mac_rewrite.sh script from the ftp server
+in-target wget {{ FTP_SERVER }}/{{ HOSTNAME }}/mac_rewrite.sh -O /usr/local/bin/mac_rewrite.sh; \
+in-target chmod 755 /usr/local/bin/mac_rewrite.sh; \
+echo "* * * * * root /usr/local/bin/mac_rewrite.sh" >> /target/etc/crontab; \
+# get the ubuntu sudo perms file from the ftp server
+in-target wget {{ FTP_SERVER }}/{{ HOSTNAME }}/90-ubuntu -O /etc/sudoers.d/90-ubuntu; \
+in-target chmod 0440 /etc/sudoers.d/90-ubuntu; \
+# setup some ssh key access
+in-target mkdir /home/ubuntu/.ssh/; \
+echo -e "{{ SSH_PUB_KEY }}" >> /target/home/ubuntu/.ssh/authorized_keys; \
+in-target chown -R ubuntu:ubuntu /home/ubuntu/.ssh/; \{% for devno in EXTRA_DASD_DEVNO %}
```

Comment:
You're talking about the jinja template at the end of like 118? I've found that placing the jinja template expressions on lines by themselves introduce an empty newline in the rendered output, breaking out of the d-i preseed/late_command string early.